src/shared/database.ts

```diff
@@ -184,7 +184,7 @@ export interface AvlTable {
     vehicle_monitoring_ref: string | null;
     line_ref: string | null;
     direction_ref: string;
-    occupancy: string | null;
+    occupancy: "full" | "seatsAvailable" | "standingAvailable" | null;
```

Comment:
Another way to do this is to create a type from `avlOccupancyValues` like:
```typescript
type AvlOccupancy = typeof avlOccupancyValues[number];
```
Which can be used like `occupancy: `AvlOccupancy | null;`

Same for occupancy further down in this file